src/store/actions.js

```diff
@@ -0,0 +1,15 @@
+import { auth } from "../firebase";
+
+const actions = {
+  AUTH_CHECK({ commit }) {
+    auth.onAuthStateChanged((user) => {
+      commit("SET_USER", user);
+    });
+  },
+
```

Comment:
This is the first of two actions

It's job is to check whether the use is authenticated. I really don't understand how the function works.

Doing a little googling, SET_USER appears to a mutation, which I haven't gotten to yet